### DISCUSSION TITLE
Gleam Node.js CLI app

### DISCUSSION THREAD

**Discussion by faveoled** [gleam-lang/gleam#2642] [Questions & support] [answered]
Hi.
What's the simplest way to jump-start a Gleam Node.js CLI project? With the ability to call some basic Node.js APIs

**Comment by lpil** [comment, marked as answer]
Hello!

If you set `target = "javascript"` in your `gleam.toml` your project will default to using JavaScript rather than Erlang.

From there you can use external functions or packages intended for Gleam on JavaScript, such as https://github.com/Crowdhailer/plinth

**Reply by faveoled** [reply]
Thanks. Also used `gleam new project_name`, `gleam add plinth`, `gleam build`. JS was in `build/dev/javascript/project_name/gleam.main.mjs`
